### ISSUE TITLE
Error in Arachni plugin

### ISSUE THREAD

**Issue by busyninja** [infobyte/faraday#381]
Please search the [Wiki](https://github.com/infobyte/faraday/wiki) for a solution before posting a ticket. Use the <strong>“New Support Request”</strong> button to the right of the screen to submit a ticket for technical support.

## Issue Type
 - Bug Re port

## Faraday version

Faraday v3.10.0 
From official .deb package 

## Component Name

Aracnhi plugin

## Steps to reproduce

Export XML report on arachni.

Try to upload on faraday-server -> manage -> import file

## Expected results

Expecting the results to appear on faraday

### Debugging tracebacks (current results)

```2020-02-03 14:09:17,309 - faraday.server.api.modules.upload_reports - DEBUG {PoolThread-twisted.internet.reactor-9} [upload_reports.py:36 - file_upload()]  Importing new plugin report in server...
2020-02-03 14:09:17,382 - faraday.server.threads.reports_processor - INFO {ReportsManager-Thread} [reports_processor.py:62 - run()]  Processing raw report /home/faraday/.faraday/uploaded_reports/56SXMXC1LKGL_dashboard_paack_co2_faraday_Arachni.xml
2020-02-03 14:09:17,383 - faraday.faraday_plugins.plugins.manager - DEBUG {ReportsManager-Thread} [manager.py:49 - _get_plugin_by_name()]  Plugin name match: arachni
2020-02-03 14:09:17,383 - faraday.faraday_plugins.plugins.manager - DEBUG {ReportsManager-Thread} [manager.py:52 - _get_plugin_by_name()]  Plugin by name Found: Arachni
2020-02-03 14:09:17,383 - faraday.server.threads.reports_processor - INFO {ReportsManager-Thread} [reports_processor.py:41 - process_report()]  Processing report [/home/faraday/.faraday/uploaded_reports/56SXMXC1LKGL_dashboard_paack_co2_faraday_Arachni.xml] with plugin [Arachni]
2020-02-03 14:09:17,396 - faraday.server.threads.reports_processor - ERROR {ReportsManager-Thread} [reports_processor.py:45 - process_report()]  Processing Error: 'NoneType' object has no attribute 'group'
2020-02-03 14:09:17,396 - faraday.server.threads.reports_processor - ERROR {ReportsManager-Thread} [reports_processor.py:46 - process_report()]  'NoneType' object has no attribute 'group'
Traceback (most recent call last):
  File "/opt/faraday/lib/python3.7/site-packages/faradaysec-3.10.0-py3.7.egg/faraday/server/threads/reports_processor.py", line 42, in process_report
    plugin.processReport(file_path)
  File "/opt/faraday/lib/python3.7/site-packages/faraday_plugins-1.0rc1-py3.7.egg/faraday_plugins/plugins/plugin.py", line 173, in processReport
    self._parse_filename(filepath)
  File "/opt/faraday/lib/python3.7/site-packages/faraday_plugins-1.0rc1-py3.7.egg/faraday_plugins/plugins/plugin.py", line 169, in _parse_filename
    self.parseOutputString(output.read())
  File "/opt/faraday/lib/python3.7/site-packages/faraday_plugins-1.0rc1-py3.7.egg/faraday_plugins/plugins/repo/arachni/plugin.py", line 398, in parseOutputString
    self.hostname = self.getHostname(parser.system.url)
  File "/opt/faraday/lib/python3.7/site-packages/faraday_plugins-1.0rc1-py3.7.egg/faraday_plugins/plugins/repo/arachni/plugin.py", line 504, in getHostname
    self.protocol = reg.group(1)
AttributeError: 'NoneType' object has no attribute 'group'```

### Screenshots

If you don't find anything on the logs, please provide screenshots of the error.

## Environment information

Fresh debian 10 minimal with faraday installed from deb package : faraday-server_amd64.deb

### Configuration files

Default install.

### Reports/Extra data

If you are having issues with plugins, please attach relevant files if possible.
(strip your reports of all sensitive information beforehand).

### OS

Provide information on your operating system. Example:

$ cat /etc/lsb-release
DISTRIB_ID=Debian
DISTRIB_RELEASE=10.2
DISTRIB_CODENAME=Buster
DISTRIB_DESCRIPTION="Debian 10.2"

**Comment by montive** [comment]
Hi there.

We'll take a look at this issue. We'll keep you updated.

Best regards,

**Comment by montive** [comment]
Hi there.

This issue has been fixed. Please, update faraday-plugins to its last version and let us know how it goes.

Cheers.

**Comment by llazzaro** [comment]
Hi,
We have released faraday v3.11 which includes the latest Arachni.
Thanks!

**Comment by busyninja** [comment]
Dear sirs,

I didn't had time until now to double check this.

Other errors are arising:

2020-05-08T20:59:13+0200 - faraday.server.threads.reports_processor - INFO {ReportsManager-Thread} [reports_processor.py:45 - process_report()]  Processing report [/home/faraday/.faraday/uploaded_reports/BJVOALJZTIPF_dashboard_paack_co_-_Full_Ultimate_-_2.xml] with plugin [Arachni]
2020-05-08T20:59:13+0200 - faraday.server.threads.reports_processor - ERROR {ReportsManager-Thread} [reports_processor.py:49 - process_report()]  Processing Error: 'NoneType' object has no attribute 'find'
2020-05-08T20:59:13+0200 - faraday.server.threads.reports_processor - ERROR {ReportsManager-Thread} [reports_processor.py:50 - process_report()]  'NoneType' object has no attribute 'find'
Traceback (most recent call last):
  File "/nix/store/h6i868qh543x9h0j0bq1jiw8inzsqam1-python3.7-faradaysec-0.1dev/lib/python3.7/site-packages/faraday/server/threads/reports_processor.py", line 46, in process_report
    plugin.processReport(file_path)
  File "/nix/store/4lggm7mgkrmni4s3xg8i13m5qk4iywar-python3.7-faraday-plugins-1.1/lib/python3.7/site-packages/faraday_plugins/plugins/plugin.py", line 253, in processReport
    self._parse_filename(filepath)
  File "/nix/store/4lggm7mgkrmni4s3xg8i13m5qk4iywar-python3.7-faraday-plugins-1.1/lib/python3.7/site-packages/faraday_plugins/plugins/plugin.py", line 249, in _parse_filename
    self.parseOutputString(output.read())
  File "/nix/store/4lggm7mgkrmni4s3xg8i13m5qk4iywar-python3.7-faraday-plugins-1.1/lib/python3.7/site-packages/faraday_plugins/plugins/repo/arachni/plugin.py", line 351, in parseOutputString
    parser = ArachniXmlParser(output)
  File "/nix/store/4lggm7mgkrmni4s3xg8i13m5qk4iywar-python3.7-faraday-plugins-1.1/lib/python3.7/site-packages/faraday_plugins/plugins/repo/arachni/plugin.py", line 34, in __init__
    self.plugins = self.getPlugins(self.tree)
  File "/nix/store/4lggm7mgkrmni4s3xg8i13m5qk4iywar-python3.7-faraday-plugins-1.1/lib/python3.7/site-packages/faraday_plugins/plugins/repo/arachni/plugin.py", line 59, in getPlugins
    return Plugins(plugins_tree)
  File "/nix/store/4lggm7mgkrmni4s3xg8i13m5qk4iywar-python3.7-faraday-plugins-1.1/lib/python3.7/site-packages/faraday_plugins/plugins/repo/arachni/plugin.py", line 244, in __init__
    self.ip = plugins_node.find('resolver').find('results').find('hostname').get('ipaddress')
AttributeError: 'NoneType' object has no attribute 'find'

Anyways we do appreciate your help in this issue, but we will not use anymore arachni for those kind of problems. Project is also discontinued.
